File: PycharmProjects/Assignment/chapter 3/excercise1.py
passes = 0
fails = 0
userinput = 0
while userinput != 1 and userinput != 2:
    userinput = int(input('Enter a number 1 or number 2: '))
    if userinput != 1 and userinput != 2:
        passes += 1
    else:
        fails += 1
        
print(passes)
print(fails)

File: PycharmProjects/jaffa/chapter 3/Excercise1.java
import java.util.Scanner;
public class Excercise1{
public static void main(String... args){
	
	Scanner scanner = new Scanner(System.in);
	
	int sumPass = 0;
	int sumFail = 0;
	
	System.out.print("Enter a number between 1 and 2: ");
	int number = scanner.nextInt();
	
	if (number != 1 && number != 2) sumFail += 1;
	else sumPass += 1;
	
	while (number != 1 && number != 2){
	System.out.print("Enter a number between 1 and 2: ");
	 number = scanner.nextInt();
	 
	if (number != 1 && number != 2) sumFail += 1;
	else sumPass += 1;
	   
	}
	System.out.printf("Total number of passes = %d", sumPass);
	System.out.printf("Total number of failures = %d", sumFail);
	
	
	
	
	
}
	
	
	
	
	
}